docs/editors/emacs.md

```diff
@@ -0,0 +1,24 @@
+# Emacs
+
+_This guide assumes that you've completed the steps described in [Prerequisites](./prerequisites.md)._
+
+1. If you're using `nix-shell`, make sure you launch your editor from the command line so it can inherit `$PATH`
+2. Install `haskell-mode` via `M-x RET package-install RET haskell-mode RET`
+3. Install `lsp-mode` via `M-x RET package-install RET lsp-mode RET`
+4. Install `lsp-haskell` via `M-x RET package-install RET lsp-haskell RET`
+5. Edit your `.emacs` file
+   ```elisp
+   (require 'lsp)
+   (require 'lsp-haskell)
+   ;; Hooks so haskell and literate haskell major modes trigger LSP setup
+   (add-hook 'haskell-mode-hook #'lsp)
+   (add-hook 'haskell-literate-mode-hook #'lsp)
+   ```
+6. Close the editor and re-open it in the project folder
+   ```bash
+   cd <your-projects-dir>/plutus-pioneer-program/code/week02
+   emacs .
```

Comment:
I think it's not really necessary to specify this path. Something like 
```
cd <your-projects-dir> emacs .
```
should be enough, I think :)

Also applies to most other entries as well.

Reply:
Thanks @Dauta , I'll fix it!

Reply:
Fixed @Dauta